fix(docs): documentation for noUnnecessaryContinue (#1164)
romefrontend/rome: website/src/docs/lint/rules/js/noUnnecessaryContinue.md

---
title: Lint Rule js/noUnnecessaryContinue
layout: layouts/rule.liquid
showHero: false

> description: disallow unnecessary continue statement inside the loop

Reviewer: ```suggestion
description: disallow unnecessary `continue` statements inside loops
```

eleventyNavigation:
	key: lint-rules/js/noUnnecessaryContinue
	parent: lint-rules
	title: js/noUnnecessaryContinue
---

# js/noUnnecessaryContinue

> `continue` statement is unnecessary as the last statement inside the loop, can be safely removed

Reviewer: ```suggestion
This rule detects `continue` statements that can be marked as unnecessary. These statements can be safely removed.
```
